<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_SelectReaction</class>
 <widget class="QDialog" name="DLG_SelectReaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Active trans.</string>
   </property>
   <widget class="QCheckBox" name="CH_AT_FY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CH_AT_TR">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Active-Link rev.</string>
   </property>
   <widget class="QCheckBox" name="CH_ALR_FX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CH_ALR_FY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FY</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Link-Cam rev.</string>
   </property>
   <widget class="QCheckBox" name="CH_LCR_FX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CH_LCR_FY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FY</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Passive trans.</string>
   </property>
   <widget class="QCheckBox" name="CH_PT_FY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CH_PT_TR">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Cam-Ground rev.</string>
   </property>
   <widget class="QCheckBox" name="CH_CGR_FY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CH_CGR_FX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FX</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Point follower</string>
   </property>
   <widget class="QCheckBox" name="CH_PF_FY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CH_PF_FX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FX</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PB_Close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
